<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>com</class>
 <widget class="QWidget" name="com">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>com</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>471</width>
     <height>258</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="comboBoxPort">
          <item>
           <property name="text">
            <string>choisisser un port série</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonOuvrirPortSerie">
          <property name="text">
           <string>Ouvrir le port série</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineEditTexteAEnvoyer"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonEnvoyer">
          <property name="text">
           <string>Envoyer</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="textEditTexteRecu"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
